_posts/2015-05-30-the-state-of-javascript.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Predictions on JavaScript in the next 12 months
+intro: Based on a recent talk I gave, in this blog post I'll discuss my thoughts on the current state of JS and what we might see over the next 12 months.
+---
+
+Recently I gave a talk at the [London JS
+Community](https://twitter.com/london_js)'s inaugural meetup titled "The State
```

Comment:
I think this could be read as meaning the meetup is called "The State of JavaScript". You could flip it so it was something like:

"Recently I gave a talk called 'The State of JavaScript' at the inaugural meetup of [London JS Community]"